honeypots/dhcp_server.py

```diff
@@ -10,25 +10,26 @@
 //  -------------------------------------------------------------
 """
 
-from twisted.internet.protocol import DatagramProtocol
-from twisted.internet import reactor
-from struct import unpack, error as StructError
+from os import getenv
 from socket import inet_aton
-from subprocess import Popen
-from os import path, getenv
+from struct import unpack, error as StructError
+from uuid import uuid4
+
+from twisted.internet import reactor
+from twisted.internet.protocol import DatagramProtocol
+
+from honeypots.base_server import BaseServer
 from honeypots.helper import (
-    close_port_wrapper,
-    get_free_port,
-    kill_server_wrapper,
     server_arguments,
     setup_logger,
     set_local_vars,
-    check_if_server_is_running,
 )
-from uuid import uuid4
 
 
-class QDHCPServer:
+class QDHCPServer(BaseServer):
+    NAME = "dhcp_server"
+    DEFAULT_PORT = 67
+
     def __init__(self, **kwargs):
```

Comment:
Use `super().__init__(**kwargs)`

Reply:
removed the `__init__` method (as it was substantially identical with that of the base class)